spring-rabbit/src/main/java/org/springframework/amqp/rabbit/log4j2/AmqpAppender.java

```diff
@@ -171,7 +174,9 @@ public static AmqpAppender createAppender(
 			@PluginAttribute("contentEncoding") String contentEncoding,
 			@PluginAttribute("clientConnectionProperties") String clientConnectionProperties,
 			@PluginAttribute("async") boolean async,
-			@PluginAttribute("charset") String charset) {
+			@PluginAttribute("charset") String charset,
+			@PluginAttribute(value = "bufferSize", defaultInt = Integer.MAX_VALUE) int bufferSize,
+			@PluginElement(BlockingQueueFactory.ELEMENT_TYPE) BlockingQueueFactory<Event> blockingQueueFactory) {
```

Comment:
Please, consider do not squash commits in the future.
That way it is easier to track the changes.
We squash them on merge any way.

More over I somehow didn't get a notification about this your change.

Reviewing now... Looks promising!

Reply:
Alright, I'll stop squashing - I'm not getting updates on this PR either, though.

Reply:
I guess something wrong was with GitHub email server yesterday.
This is not only PR without notification though.